assets/css/settings-styles.css

```diff
@@ -1152,6 +1152,81 @@ div.upgrade-table-description a,
 	color: #fcfbf7;
 }
 
+.card-container {
+    border-radius: 5px;
+    width: calc(100% - 30px);
+    display: flex;
+    flex-wrap: wrap;
+    gap: 30px;
+    padding: 15px;
+    background-color: #f0f0f0;
```

Comment:
You can use `#f1e9fc` for the background color of the section. Which is a pale tint variation of the new primary brand color `#6e1edc`.